.github/workflows/handle_release_tag.yaml

```diff
@@ -0,0 +1,34 @@
+name: Handle Release Tag
+
+on:
+  push:
+    tags:
+    - 'v*'
+
+jobs:
+  create_release:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v4
+      with:
+        submodules: true
+        fetch-tags: true
+        fetch-depth: 0
+    - name: Repair tag
+      run: git fetch -f origin ${{ github.ref }}:${{ github.ref }}
+    - name: Verify that the tag is annotated
+      run: if test x$(git for-each-ref ${{ github.ref }} | awk '{print $2}') = xtag; then /bin/true; else echo "\"${{ github.ref }}\" does not look like an annotated tag!"; /bin/false; fi
```

Comment:
I think you should use `run: |` and make this multiline so that it's easier to read

Reply:
This is taken from the original.  I'd rather leave it rather then risk breaking it.

Reply:
Works for me, then